### ISSUE TITLE
Add Server Side Variable to store API information for Tide/Weather service

### ISSUE THREAD

**Issue by Fish360** [Fish360/Journal#56]

**Comment by Fish360** [comment]
@jannunzi 

Per our conversation, I have created this issue.  Please add more details as needed.  In short, we need to de-couple the API service connection information. This will allow for us to quickly change the API service (if and when required).

**Comment by Fish360** [comment]
@patelankita 

This may impact your work.

**Comment by Fish360** [comment]
@yashsnehalshah 

Yash,

Can you take this one on?

@jannunzi

**Comment by yashsnehalshah** [comment]
@Fish360 

Yes surely Julio. I need some more understanding of the requirement and the problem before i get my hands on the code(which you have already asked to Professor). The API information, as of now, is stored and available on both client side services and server side services(for example: weatheronline.service.client.js and world-weather-online.service.server.js). So it will be really helpful if i get a little more insight on what is the end requirement so that i can manipulate the flow of client-service-server accordingly.

**Comment by yashsnehalshah** [comment]
@Fish360 

Hi Julio,

Just for your kind information, i may not be very active in the upcoming week due to a presentation and final exam in another course. Hope that is ok. I will surely get back to work as soon as i get free from that stuff. Meanwhile may i please request you to identify most high priority issues that I and Ankita can work on before we wrap up in some weeks?

Thanks and regards,

Yash

**Comment by yashsnehalshah** [comment]
@Fish360 

Hi Julio, just got over from this tiresome week of presentation and final exam. Getting back to work now. Please let me know the issues that you want me to work on. I will also work on this issue as soon as i get more clarification/details of what needs to be achieved.

Thanks and regards,

Yash
